src/opm/parser/eclipse/Parser/raw/RawKeyword.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+  Copyright 2013 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef RAWKEYWORD_HPP
+#define RAWKEYWORD_HPP
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <list>
+
+#include "RawEnums.hpp"
+#include "RawConsts.hpp"
+
+namespace Opm {
+
+    class RawRecord;
+    class string_view;
+
+    class RawKeyword {
+    public:
+        RawKeyword(const std::string& name, const std::string& filename, std::size_t lineNR, bool raw_string, Raw::KeywordSizeEnum sizeType);
```

Comment:
You need to `#include <cstddef>` to use `std::size_t` in a function declaration.